### ISSUE TITLE
TibiaWiki issues

### ISSUE THREAD

**Issue by benjaminkomen** [Mytherin/Tibialyzer#9]
I noticed in [parseXML.py](https://github.com/Mytherin/Tibialyzer/blob/master/Database%20Scan/parseXML.py) you are fixing some typos. It would be cool for TibiaWiki if you compile a list of mistakes, which we can fix, instead of having to bypass them yourself. Also, if any unknown (=not on the wiki) creatures, items, etc. are encountered it would be useful to know that as well.

Alvast bedankt.
Benjamin

**Comment by Mytherin** [comment]
Most of the 'fixes' I have in that file are because of my laziness. They function on tibia wikia (e.g. "bone beast" redirects to "bonebeast"), but since my script does not deal with redirects and there were only a few of these issues I just hardcoded the redirects in rather than deal with them properly. I should fix that at some point, but this was easier for the time being.

If any unknown creatures are encountered I wouldn't know it myself unless a user reports it to me. However, I have now added a feature so Tibialyzer automatically saves all the users' loot messages to a file. I will likely add in a feature so users can send these to me, then looking at those files I could find unknown creatures.
